wordpress-mobile/WordPress-Android
# RI:  Choose Interests - Introduce loading state

Master Issue: #12027

This PR introduces loading state into the Choose Interests screen.

### To test

- Launch app and go to Reader tab
- Notice that progress bar is shown on a blank screen
- Wait for a while
- Notice that 
    - Progress bar is hidden
    - Done button with text "Select a few to continue" is shown in disabled state
    - Interest chips are displayed along with title, subtitle 

#### Note

This PR includes UI changes from PR: Choose Interests UI #12110. These are still being finalised (like whether Done button should be scrollable in the landscape mode), so please ignore them in this PR review.

### Reviewing
Only 1 reviewer is needed but anyone can review.

PR submission checklist:
